rllib/algorithms/algorithm.py

```diff
@@ -3052,9 +3063,8 @@ def _run_one_training_iteration(self) -> Tuple[ResultDict, "TrainIterCtx"]:
                 key=AGGREGATOR_ACTOR_RESULTS,
             )
 
-        # Only here (at the end of the iteration), reduce the results into a single
-        # result dict.
-        return self.metrics.reduce(), train_iter_ctx
+        # Only here (at the end of the iteration), compile the results into a single result dict.
```

Comment:
nit: Add one-liner comment on what compile does differently from reduce: It adds throughput metrics and returns values, not `Stats` objects, correct?